src/test/groovy/de/dkfz/roddy/execution/jobs/cluster/pbs/PBSJobManagerSpec.groovy

```diff
@@ -87,11 +87,11 @@ class PBSJobManagerSpec extends Specification {
         def parms = JobManagerOptions.create().build()
         TestExecutionService testExecutionService = new TestExecutionService("test", "test")
         PBSJobManager jm = new PBSJobManager(testExecutionService, parms)
-        Method method = PBSJobManager.class.getDeclaredMethod("processQstatOutput", List)
+        Method method = PBSJobManager.class.getDeclaredMethod("processQstatOutput", String)
```

Comment:
Why do you use reflection here? Could you also make the method access public / package based?

Reply:
I removed the reflection